src/worker.ts

```diff
@@ -1,45 +1,38 @@
 import {
-	addMissedUpdatesRole,
-	callDiscordNicknameBatchUpdate,
-	syncExternalAccounts,
-	syncIdle7dUsers,
-	syncIdleUsers,
-	syncOnboarding31dPlusUsers,
-	syncUnverifiedUsers,
-	syncUsersStatus,
+	addMissedUpdatesRoleHandler,
+	callDiscordNicknameBatchUpdateHandler,
+	syncExternalAccountsHandler,
+	syncIdle7dUsersHandler,
+	syncIdleUsersHandler,
+	syncOnboarding31dPlusUsersHandler,
+	syncUnverifiedUsersHandler,
+	syncUsersStatusHandler,
 } from './handlers/scheduledEventHandler';
 import { env } from './types/global.types';
 
-const EVERY_6_HOURS = '0 */6 * * *';
 const EVERY_11_HOURS = '0 */11 * * *';
 const EVERY_20_MINUTES = '*/20 * * * *';
-const EVERY_30_MINUTES = '*/30 * * * *';
 
 export default {
 	// eslint-disable-next-line no-unused-vars
 	async scheduled(req: ScheduledController, env: env, ctx: ExecutionContext) {
 		switch (req.cron) {
-			case EVERY_6_HOURS: {
-				return await callDiscordNicknameBatchUpdate(env);
-			}
 			case EVERY_11_HOURS: {
-				return await addMissedUpdatesRole(env);
-			}
-
-			case EVERY_20_MINUTES: {
-				await syncIdleUsers(env);
-				// await syncNickNames(env); TODO: Enable it once changes from website-backend is merged
-				await syncIdle7dUsers(env);
-				await syncOnboarding31dPlusUsers(env);
-				console.log('Worker for syncing idle users, nicknames, idle 7d users, and onboarding 31d+ users has completed.');
+				callDiscordNicknameBatchUpdateHandler(env);
+				addMissedUpdatesRoleHandler(env);
 				break;
 			}
 
-			case EVERY_30_MINUTES: {
-				await syncUsersStatus(env);
-				await syncExternalAccounts(env);
-				await syncUnverifiedUsers(env);
-				console.log('Worker for syncing user status, external accounts, and unverified users has completed.');
+			case EVERY_20_MINUTES: {
+				syncIdleUsersHandler(env);
+				//  syncNickNamesHandler(env); TODO: Enable it once changes from website-backend is merged
+				syncIdle7dUsersHandler(env);
+				syncOnboarding31dPlusUsersHandler(env);
+				syncUsersStatusHandler(env);
+				syncExternalAccountsHandler(env);
+				syncUnverifiedUsersHandler(env);
+				console.log(`Worker for syncing idle users, nicknames, idle 7d users, and onboarding 31d+ users has completed.
+				Worker for syncing user status, external accounts, and unverified users has completed.`);
```

Comment:
Will running all these together cause discord API rate limiting ?

Reply:
The global rate limiting on discord API is 50 requests per second and for our use case we are making 6 API requests indirectly

Reply:
Can you please share the doc reference for the rate-limiting info

But when calling this API won't they call the discord API, so when all these API calls discord API then won't we hit 50 requests per second

Reply:
https://discord.com/developers/docs/topics/rate-limits#global-rate-limit
I don't think we would hit 50 times per second with a trigger from the corn job alone 
Let's discuss